src/plugins/discover/public/application/components/sidebar/discover_field.test.tsx

```diff
@@ -104,9 +104,4 @@ describe('discover sidebar field', function () {
     findTestSubject(comp, 'fieldToggle-bytes').simulate('click');
     expect(props.onRemoveField).toHaveBeenCalledWith('bytes');
   });
-  it('should trigger onShowDetails', function () {
-    const { comp, props } = getComponent();
-    findTestSubject(comp, 'field-bytes-showDetails').simulate('click');
-    expect(props.onShowDetails).toHaveBeenCalledWith(true, props.field);
-  });
```

Comment:
With a few adaptions this test should work

```Typescript
it('should trigger getDetails', async () => {
    const { comp, props } = getComponent();
    findTestSubject(comp, 'field-bytes-showDetails').simulate('click');
    expect(props.getDetails).toHaveBeenCalledWith(props.field);
  });
```
you need to adapt line 83 in the following way:
```Typescript
getDetails: jest.fn(() => ({ buckets: [] })),
```
Then it should work

Reply:
@kertal to follow up with a PR adding back this test as I had some trouble doing that.